app/views/users/login.erb

```diff
@@ -4,13 +4,12 @@
   <span><%= @errors %></span>
 <% end %>
 
-<div class="form-group">
-  <form method="post" action="/users/login">
-    <label for="email">Email:</label>
-    <input type="email" name="email" value="<%= @email %>">
-    <label for="password">Password:</label>
-    <input type="password" name= "password">
-    <input type="submit" value="Login" class="btn btn-default">
-  </form>
-</div>
+<form class="form-group">
+<form method="post" action="/users/login">
```

Comment:
This should probably be a `<div class="form-group">` inside of the form instead of the double form tag. @kmbradshaw